hercules/emulator.py

```diff
@@ -266,7 +266,7 @@ def receive_amrwind_data(self):
         print("AMRWindSpeed:", wind_speed_amr_wind)
         print("AMRWindDirection:", wind_direction_amr_wind)
         print("AMRWindTurbinePowers:", turbine_power_array)
-        print(" AMRWIND number of turbines here: ", self.num_turbines)
+        print("AMRWIND number of turbines here: ", self.num_turbines)
```

Comment:
Perhaps this line can simply be removed? It seems pretty redundant.

Reply:
I agree it probably doesn't need to be printed every time step. Deleted.